src/main/java/school/lemon/changerequest/java/Accountant.java

```diff
@@ -0,0 +1,28 @@
+package school.lemon.changerequest.java;
+
+
+/**
+ * Created by lera on 15.12.16.
+ */
+public class Accountant extends Employee {
```

Comment:
My mistake.
You have override `calculateSalary` in Manager, while there was no need.
You implementation in Employee match both Accountant and Manager, so you should remove your Manager implementation.